src/commands/typegen.ts

```
@@ -61,7 +61,8 @@ export async function action(commandOptions: TypegenOptions = {}): Promise<void>
 	const startTime = performance.now();
 	const spinner = createSpinner("Fetching Rojo sourcemap...");
 
-	const rojoSourceMap = await getRojoSourceMap(commandOptions.project);
+	const projectPath = commandOptions.project ?? config.rojoProjectPath;
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)